torch/jit/__init__.py

```diff
@@ -533,11 +533,13 @@ class ScriptMeta(type(torch._C.ScriptModule)):
     # a pybind11 type
     def __init__(cls, name, bases, attrs):
         # find all the script methods
+        cls.original_methods = {}
```

Comment:
Can you call this `cls.__original_methods`? Otherwise it might clash with an attribute that someone has defined in their class.

Reply:
done